src/main.cpp

```diff
@@ -355,14 +363,18 @@ void getCommandSchemaJson(JsonVariant json)
 
   JsonObject colour = channelProperties.createNestedObject("colour");
   colour["type"] = "array";
+  colour["description"] = "Set the brightness for each channel (value between 0 - 255)";
   colour["minItems"] = 1;
   colour["maxItems"] = MAX_LED_COUNT;
   JsonObject colourItems = colour.createNestedObject("items");
   colourItems["type"] = "integer";
+  colourItems["minimum"] = 0;
+  colourItems["maximum"] = 255;
 
   JsonObject fadeIntervalUs = channelProperties.createNestedObject("fadeIntervalUs");
   fadeIntervalUs["type"] = "integer";
   fadeIntervalUs["minimum"] = 0;
+  fadeIntervalUs["description"] = "Time to fade from off to on, or vice versa, in milliseconds (default is 500ms)";
```

Comment:
"Time to fade from off -> on (or vice versa), in microseconds (defaults to 500us)"